abm/app.py

```diff
@@ -73,19 +73,19 @@ def start_headless():
 def start_playground():
     # changing env file according to playground default parameters before
     # running any component of the SW
-    save_isims_env()
+    save_isims_env(root_abm_dir, EXP_NAME, pgt, envconf)
     # Start interactive simulation
     sim = PlaygroundSimulation()
     sim.start()
 
 
-def save_isims_env():
+def save_isims_env(env_dir, _EXP_NAME, _pgt, _envconf):
     """translating a default parameters dictionary to an environment
     file and using env variable keys instead of class attribute names"""
```

Comment:
In case we now have function parameters we should include these in the docstring